app/renderer/components/Inspector/Screenshot.js

```diff
@@ -89,20 +137,48 @@ export default class Screenshot extends Component {
     // If we're tapping or swiping, show the 'touch' cursor style
     const screenshotStyle = {};
     if (screenshotInteractionMode === 'tap' || screenshotInteractionMode === 'swipe') {
-      screenshotStyle.cursor = 'crosshair'; // TODO: Change this to touch
+      screenshotStyle.cursor = 'crosshair';
     }
 
     recursive(source);
 
     // Show the screenshot and highlighter rects. Show loading indicator if a method call is in progress.
     return <Spin size='large' spinning={!!methodCallInProgress}>
```

Comment:
there is now enough HTML code here I wonder if we should factor out some smaller components. up to you.